### PR TITLE
Boil down first CI tutorial

### PR DESCRIPTION
**Pull request by sdruskat**
Tihs PR prepares a working tutorial for using GitHub Zenodo Sandbox publication via a template.

### CODE REVIEW

**Comment by sdruskat** [comment]
Shoudl add a link to *concept DOI* in the last paragraph, pointing to Zenodo's "versioning DOIs" docs.

**Comment by sdruskat** [comment]
> Shoudl add a link to _concept DOI_ in the last paragraph, pointing to Zenodo's "versioning DOIs" docs.

Done in https://github.com/hermes-hmc/workflow/pull/180/commits/a5c2b08cfe0be465ec0fbfc7ff793a1e672c7d80.
Ready for review now.

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (jkelling) [inline docs/source/tutorials/automated-publication-with-ci.md:65]: I think we really need a decision on the executions order/merge order https://github.com/hermes-hmc/workflow/issues/156 , because with the current implementation this tutorial suggests to prioritize information harvested from git over that manually entered meta data from cff.

If we keep the current ordering, a more sensible suggestion would be:
```toml
form = ["git", "cff"]
```

**Review by jkelling — APPROVED** [review]
Looks good, apart from the `["cff, "git"]` harvest order.

Does the folder `.hermes/curate` remain in the repo after the publication PR is merged?

**Review by poikilotherm — APPROVED** [review]
I took the liberty of adding some more subtle changes. Hope this is to your liking, especially the workflow diagram. Tune as you see fit - otherwise good to go!

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (poikilotherm) [inline docs/source/tutorials/automated-publication-with-ci.md:69]: @sdruskat this is missing the post-processing configuration, which needs to be straightened out a little over in #175

Comment (poikilotherm) [reply docs/source/tutorials/automated-publication-with-ci.md:69]:
```suggestion
[postprocess]
execute = [ "config_record_id" ]

```

Comment (sdruskat) [reply docs/source/tutorials/automated-publication-with-ci.md:69]: Done in https://github.com/hermes-hmc/workflow/pull/180/commits/0b056d4ee6244c94f764f8a98b37307ae4f9dec2

**Comment by sdruskat** [comment]
> I took the liberty of adding some more subtle changes. Hope this is to your liking, especially the workflow diagram. Tune as you see fit - otherwise good to go!

Cheers, I finetuned a bit. Will merge once the CI build is done.

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (sdruskat) [reply docs/source/tutorials/automated-publication-with-ci.md]: Done in https://github.com/hermes-hmc/workflow/pull/180/commits/e18fadcf74059a30318c4fab4c5e08df0b74ce56

Comment (led02) [reply docs/source/tutorials/automated-publication-with-ci.md]: I would prefer to simply keep out the `git` for now as it does not provide to much usable output based on uncertain semantics...

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (led02) [inline docs/source/tutorials/automated-publication-with-ci.md:71]: Is there a good reason to switch off CFF validation here?

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (led02) [inline docs/source/tutorials/automated-publication-with-ci.md:111]: Maybe this is just a litte too lazy. I would exprect at least some synopsis what and why to adapt...

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (led02) [inline docs/source/tutorials/automated-publication-with-ci.md:120]: This is not true: We create a new PR that needs to be closed before the publication is done (i.e., "curation step").

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (led02) [inline docs/source/tutorials/automated-publication-with-ci.md:157]: Again: No, it only creates the curation PR that needs to be merged before the deposition takes place.

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (led02) [inline docs/source/tutorials/automated-publication-with-ci.md]: This reads rather cryptic... it's not the *updates from the publication* but the postprocessing that changes stuff...

**Review by led02 — COMMENTED** [review]
**note:** I found some inconsistencies. This might be due to unclear wording. Yet, it does not correctly describe the process.

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (sdruskat) [reply docs/source/tutorials/automated-publication-with-ci.md:71]: Only that it was copied from showcase (which I don't think has "version" perhaps?). Happy to remove.

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (sdruskat) [reply docs/source/tutorials/automated-publication-with-ci.md:111]: Didn't want to overburden the tutorial for this. Once we have a cleaner template (e.g. based on Marketplace Action), we can pull this in here.

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (sdruskat) [reply docs/source/tutorials/automated-publication-with-ci.md:120]: :+1:

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (sdruskat) [reply docs/source/tutorials/automated-publication-with-ci.md:157]: :+1:

File: docs/source/tutorials/automated-publication-with-ci.md
Comment (sdruskat) [reply docs/source/tutorials/automated-publication-with-ci.md]: Okay, will adapt.

**Pull request merged**
